### ISSUE TITLE
fix(html-spec): add touch event handler attributes (ontouchstart, etc.)

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3660]
## Describe the bug

Touch event handler attributes (`ontouchstart`, `ontouchend`, `ontouchmove`, `ontouchcancel`) are reported as disallowed by the `invalid-attr` rule.

## Code Example

```html
<div ontouchstart="handler()">touch me</div>
```

## The current behavior

```
The "ontouchstart" attribute is disallowed. Did you mean "onloadstart"?
```

## The expected behavior

No violation. These are valid event handler attributes.

## Impact

4 false positives in the nu-html-checker compatibility benchmark.

## Affected package

`@markuplint/html-spec` — global event handler attributes
